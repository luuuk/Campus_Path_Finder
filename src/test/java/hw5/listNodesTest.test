# This test is for the getNode and addNode methods

# Create a new Graph, g
CreateGraph g

# List nodes in g
ListNodes g

# Add node n1 to g
AddNode g n1

# Add node n2 to g
AddNode g n2

# List nodes in g
ListNodes g