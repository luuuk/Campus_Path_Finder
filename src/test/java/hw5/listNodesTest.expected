# This test is for the getNode and addNode methods

# Create a new Graph, g
created graph g

# List nodes in g
g contains:

# Add node n1 to g
added node n1 to g

# Add node n2 to g
added node n2 to g

# List nodes in g
g contains: n1 n2